src/borda_clj/client.clj

```diff
@@ -0,0 +1,90 @@
+(ns borda-clj.client
+  (:use [slingshot.slingshot :only [throw+ try+]])
+  (:require [aleph.http :as http]
+            [byte-streams :as bs]
+            [cheshire.core :as json]))
+
+(def submit-key {:op "_submit"})
+
+(defn submit-failed [measurements values]
+  (update measurements submit-key (partial merge-with +) {:error_count (get values :_submits)}))
```

Comment:
I don't understand when would `(get values :_submits)` ever be anything different from 1.

Reply:
> I don't understand when would (get values :_submits) ever be anything different from 1.

Oh, good point. My logic [here](https://github.com/getlantern/borda-clj/pull/1/files#diff-00f5ab632e4080f183618ccb8ef5a023R16) is broken. It's supposed to add 1 to the submits every time we submit a new measurement with a key matching an existing.

Reply:
Oh, never mind, logic should be fine. Submits will get merged like all other values [here](https://github.com/getlantern/borda-clj/pull/1/files#diff-00f5ab632e4080f183618ccb8ef5a023R19).